test/blackbox-tests/test-cases/install/dune-package-source-path.t

```diff
@@ -89,6 +87,8 @@ Test paths on public libraries with `.` are correct
     "_build/install/default/lib/b/sub/b__Foo.cmi" {"sub/b__Foo.cmi"}
     "_build/install/default/lib/b/sub/b__Foo.cmt" {"sub/b__Foo.cmt"}
     "_build/install/default/lib/b/sub/b__Foo.cmx" {"sub/b__Foo.cmx"}
+    "_build/install/default/lib/b/sub/child/bar.ml" {"sub/child/bar.ml"}
+    "_build/install/default/lib/b/sub/child/child.ml" {"sub/child/child.ml"}
```

Comment:
So the values in the dune-package file are correct?

Reply:
I believe they are. Check the last line of this test.